### DISCUSSION TITLE
Removing Wazuh agents error

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#5500] [Unsupported Versions]
Hi, 

Getting some strange errors while tidying up our wazuh agents via `so-wazuh-agent-manage`. some went through fine then some started dropping this error
```
Provide the ID of the agent to be removed (or '\q' to quit): 044
Confirm deleting it?(y/n): y
2021/09/14 13:47:22 manage_agents: INFO: Cannot connect to '/queue/db/wdb': Connection refused (111). Waiting 1 seconds to reconnect.
2021/09/14 13:47:23 manage_agents: INFO: Cannot connect to '/queue/db/wdb': Connection refused (111). Waiting 2 seconds to reconnect.
2021/09/14 13:47:25 manage_agents: INFO: Cannot connect to '/queue/db/wdb': Connection refused (111). Waiting 3 seconds to reconnect.
2021/09/14 13:47:28 manage_agents: ERROR: Unable to connect to socket '/queue/db/wdb'.
2021/09/14 13:47:28 manage_agents: ERROR: Unable to connect to socket '/queue/db/wdb'.
2021/09/14 13:47:28 manage_agents: ERROR: Could not remove the DB of the agent 044. Error: -2.
Agent '044' removed.
```
it looks to remove fine as it no longer appears when listing agents but the "could not remove the db of agent 044" makes me think that it hasn't moved everything and could cause issues if 044 gets re-used. Is this something to look into further or just ignore?

**Comment by dougburks** [comment]
I would think you could just ignore it for now. If you wanted to try something else, you could try removing it again or adding a new agent and then removing that new agent.
